_i18n/en.yml

```diff
@@ -147,30 +147,30 @@ merchants:
   use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.
   dirdescr: "If you are looking for places to spend your Monero, the directories below have some options. Please remember to do your own research before trusting them with your money:"
   descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private than their centralized counterparts and allow people to be in complete control of their coins if they are non-custodial, but they usually have less volume than 'traditional' exchanges. In the section below, we offer a list of P2P exchanges that support Monero.
-  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="https://www.getmonero.org/2021/08/20/atomic-swaps.html">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post). The technology is brand new, so there aren't many swap providers yet, but you can already sell XMR for BTC using this protocol at
+  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="https://www.getmonero.org/2021/08/20/atomic-swaps.html">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post).
```

Comment:
I think this line should be removed or moved to `unstoppableswapdesc`, otherwise there would be a duplicate mention.

Reply:
Deleting the entire `swapdesc` should be ok. The atomic swap blog could still need to be linked. We can see how it looks without `swapdesc` first

Reply:
The "best" way to swap is Atomic Swaps.. no third party. Peer to peer. Trustless.
- Deleting this swapdesc entirely
- edit the 1st paragraph to mention atomic swaps being the "best" and p2p being the 2nd "best" (although best seems subjective),
Thoughts?